src/state/PieceBoss.js

```diff
@@ -0,0 +1,17 @@
+import { TYPES } from "./BoardBoss";
+
+let id = 0;
```

Comment:
V important. When I was sketching out the `update` loop, I ran into the issue of not having any unique identifier to use to update the x,y for the graphic.